recipes/podofo/all/conanfile.py

```diff
@@ -0,0 +1,87 @@
+    homepage = "http://podofo.sourceforge.net"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "PoDoFo is a library to work with the PDF file format."
+    topics = ("conan", "PDF", "PoDoFo", "podofo")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "cmake", "cmake_find_package"
+    exports_sources = "CMakeLists.txt"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
+
+        if self.settings.os == "Macos" and self.options.shared:
+            raise ConanInvalidConfiguration("PoDoFo doesn't support shared libraries on MacOS")
+
+        if self.settings.compiler == "gcc" and tools.Version(self.settings.compiler.version) < "5":
+            raise ConanInvalidConfiguration("PoDoFo requires GCC >=5")
+
+    def requirements(self):
+        self.requires("freetype/2.10.4")
+        if str(self.settings.os) in ["Linux", "Macos"]:
+            self.requires("fontconfig/2.13.92")
+        self.requires("libjpeg/9d")
+        self.requires("libunistring/0.9.10")
+        self.requires("libtiff/4.1.0"),
+        self.requires("libidn/1.36")
+        self.requires("openssl/1.1.1h")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("podofo-{0}".format(self.version), self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.definitions["PODOFO_BUILD_LIB_ONLY"] = True
+        self._cmake.definitions["PODOFO_BUILD_SHARED"] = self.options.shared
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy("COPYING", src=self._source_subfolder, dst="licenses")
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+    def package_info(self):
+        self.cpp_info.filenames["cmake_find_package"] = "podofo"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "podofo"
+        self.cpp_info.names["pkg_config"] = "libpodofo-0"
+        self.cpp_info.names["cmake_find_package"] = "podofo"
+        self.cpp_info.names["cmake_find_package_multi"] = "podofo"
```

Comment:
Where do these come from?

Checking the source code they are not installed by the project?

Reply:
Good question :)
```
self.cpp_info.names["pkg_config"] = "libpodofo-0"
``` 
was suggested by https://github.com/conan-io/conan-center-index/pull/3594#discussion_r525911110

Reply:
pkg-config file is listed https://github.com/conan-io/conan-center-index/pull/3594#issuecomment-729518459 hence my suggestion. for cmake files I have no idea.

Reply:
about cmake files I was suggesting other recipes.
for this
```
        self.cpp_info.names["cmake_find_package"] = "podofo"
        self.cpp_info.names["cmake_find_package_multi"] = "podofo"
```
Should I change to `self.cpp_info.name = "podofo"` ?
For filenames I can't find in [documentation](https://docs.conan.io/en/latest/reference/conanfile/methods.html?highlight=package_info#package-info)

Reply:
Please not, the cci recipes keep `self.cpp_info.name` unmodified.

Reply:
Ok, I've found about cpp_info https://docs.conan.io/en/latest/reference/conanfile/attributes.html#cpp-info.
As I undersand `self.cpp_info.{file}names` could be, but there is no need, because I'm providing the same name as recipe name so it won't be changed.

Reply:
If podofo doesn't have official CMake module or config files, you should remove the lines related to cmake_find_package and cmake_find_package_multi.
By the way podofo will be the default name since it's the name of the recipe. Moreover using `filenames` requires conan 1.28, so keeping this line raises the minimum conan version required while it's unecessary.

Reply:
These lines was deleted in ths commit https://github.com/conan-io/conan-center-index/pull/3594/commits/53023a170ebf1792b799cdc22e491d93a2a9d225

Reply:
I see that a cmake folder is deleted: https://github.com/nicraMarcin/conan-center-index/blob/2bebc0c30a342249d535eb02e1572b3019770d85/recipes/podofo/all/conanfile.py#L77

So what's the content of those files?